src/App.tsx

```diff
@@ -27,6 +27,8 @@ const Gateways = React.lazy(() => import('./pages/Gateways'));
 const Gateway = React.lazy(() => import('./pages/Gateway'));
 const Staking = React.lazy(() => import('./pages/Staking'));
 const Observers = React.lazy(() => import('./pages/Observers'));
+const BalancesMain = React.lazy(() => import('./pages/Balances'));
+const Balances = React.lazy(() => import('./pages/Balances/Balances'));
```

Comment:
Lil confusing, maybe like "AllBalances" and "UserBalances", or "Balances" and "BalancesForAddress" or something